apps/ledger-live-desktop/scripts/resolver.js

```diff
@@ -19,6 +19,14 @@ module.exports = (path, options) => {
         "@firebase/util",
         "firebase",
         "uuid",
+        "rpc-websockets",
```

Comment:
Why do we need to do this, please?

Reply:
The tests are not working without this change after updating solana libs, related to a recent change to make the rpc-websockets lib esm compatible IIRC